packages/engine/src/renderer/WebGLRendererSystem.ts

```diff
@@ -233,29 +236,12 @@ export class EngineRenderer {
     dispatchAction(Engine.store, EngineRendererAction.setQualityLevel(state.qualityLevel.value))
     dispatchAction(Engine.store, EngineRendererAction.setPostProcessing(state.qualityLevel.value > 2))
   }
-
-  async loadGraphicsSettingsFromStorage() {
-    const [automatic, qualityLevel, useShadows, /* pbr, */ usePostProcessing] = await Promise.all([
-      ClientStorage.get(databasePrefix + RENDERER_SETTINGS.AUTOMATIC) as Promise<boolean>,
-      ClientStorage.get(databasePrefix + RENDERER_SETTINGS.QUALITY_LEVEL) as Promise<number>,
-      ClientStorage.get(databasePrefix + RENDERER_SETTINGS.USE_SHADOWS) as Promise<boolean>,
-      // ClientStorage.get(databasePrefix + RENDERER_SETTINGS.PBR) as Promise<boolean>,
-      ClientStorage.get(databasePrefix + RENDERER_SETTINGS.POST_PROCESSING) as Promise<boolean>
-    ])
-    dispatchAction(Engine.store, EngineRendererAction.setAutomatic(automatic ?? true))
-    dispatchAction(Engine.store, EngineRendererAction.setQualityLevel(qualityLevel ?? 1))
-    dispatchAction(Engine.store, EngineRendererAction.setShadows(useShadows ?? true))
-    // dispatchAction(Engine.store, EngineRendererAction.setPBR(pbr ?? true))
-    dispatchAction(Engine.store, EngineRendererAction.setPostProcessing(usePostProcessing ?? true))
-  }
 }
 
 export default async function WebGLRendererSystem(world: World) {
   new EngineRenderer()
 
-  receiveActionOnce(Engine.store, EngineEvents.EVENTS.JOINED_WORLD, () =>
-    EngineRenderer.instance.loadGraphicsSettingsFromStorage()
-  )
+  receiveActionOnce(Engine.store, EngineEvents.EVENTS.JOINED_WORLD, async () => await restoreEngineRendererData())
```

Comment:
no need to make this async, the function will be called synchronously anyway, and this restore function does not need to be awaited